opmd_viewer/addons/pic/lpa_diagnostics.py

```diff
@@ -17,4 +20,668 @@ def __init__(self, path_to_dir):
             one file per iteration, and the name of the files should end
             with the iteration number, followed by '.h5' (e.g. data0005000.h5)
         """
-        OpenPMDTimeSeries.__init__( path_to_dir )
+        super(LpaDiagnostics, self).__init__( path_to_dir )
```

Comment:
What about doing
```python
OpenPMDTimeSeries.__init__( self, path_to_dir )
```
instead of 
```python
super(LpaDiagnostics, self).__init__( path_to_dir )
```
as it is more transparent. 

(I forgot the `self` in the previous version of the code; sorry)